share/spice/detect_lang/detect_lang.js

```diff
@@ -108,42 +87,82 @@ function ddg_spice_detect_lang (api_result) {
 
         return langs[language] || "";
     };
-    Handlebars.registerHelper("DetectLang_expandLang", expandLang);
 
-    if(expandLang(api_result.data.detections[0].language) === "") {
-        return;
+    var formatTitle = function (first) {
+        var title = "This text is";
+        if (first.isReliable) {
+            title += " definitely ";
+        } else {
+            title += " probably ";
+        }
+        title += expandLang(first.language);
+        title += " (" + getPercentConfidence(first.confidence) + ").";
+        return title;
     }
 
-    var d0 = api_result.data.detections[0],
-        d1 = null;
-
-    if (api_result.data.detections.length > 1) {
-        d1 = api_result.data.detections[1];
+    var formatSubtitle = function (second) {
+        var subtitle = "";
+        if (!second) {
```

Comment:
nit: if you make the check `if (second) {` you'll only need 1 `return` statement:

```js
var subtitle = "";
if (second) {
  subtitle += "...";
}
return subtitle;
```